kolibri-electron/src/index.js

```diff
@@ -138,6 +138,7 @@ async function checkVersion() {
   console.log(`${kolibriHomeVersion} < ${pluginVersion}`);
   if (kolibriHomeVersion < pluginVersion) {
     console.log('Newer version, replace the .endless-key directory and cleaning cache');
+    await fsExtra.remove(KOLIBRI_HOME);
```

Comment:
:+1: this should do the trick. But then why the rename is needed?

Reply:
This fixes the issue, the rename is done in a second commit because just with the replace, the old endless key is not working anymore. @dylanmccall left a comment on slack about the the different directory solution and I though it could be a good thing to do.

We've the discussion in slack about it:
 * https://endlessos.slack.com/archives/C01B4AAG2S1/p1645600044792049
 * https://endlessos.slack.com/archives/C01B4AAG2S1/p1645611851103279

Reply:
OK sounds good. Can you please move the "iguana" to a constant? And add a note in the README for updating it on each release?